terminus-ui/logo/src/logo.component.md

```diff
@@ -0,0 +1,67 @@
+<h1>Logo</h1>
+
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Basic usage](#basic-usage)
+- [Type](#type)
+- [Colors](#colors)
+- [Sizing](#sizing)
+- [Special Cases](#special-cases)
+- [Available](#available)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+
+## Basic usage
+
+The most basic implementation is only html. Defaults to full gradient type. 
+
+```html
+<ts-logo></ts-logo>
+```
+
+## Type
+Types are full logo or mark, gradient (full color) or solid.
+
+```html
+<ts-logo type="full-solid"></ts-logo>
+<ts-logo type="mark-gradient"></ts-logo>
+```
+
+Search `TS_LOGO_TYPES` to see all allowed types.
+
+## Colors
+
+Solid logos are white by default, but can also be black (Terminus Dark) or gray (#44525E).
```

Comment:
I'd prefer to not reference the exact hex color here as there is a high chance it won't get updated if/when the logo color is.